stdlib/SuiteSparse/src/cholmod.jl

```diff
@@ -1714,7 +1714,7 @@ end
 (\)(L::Factor{T}, B::StridedMatrix) where {T<:VTypes} = Matrix(L\Dense{T}(B))
 (\)(L::Factor, B::Sparse) = spsolve(CHOLMOD_A, L, B)
 # When right hand side is sparse, we have to ensure that the rhs is not marked as symmetric.
-(\)(L::Factor, B::SparseVecOrMat) = sparse(spsolve(CHOLMOD_A, L, Sparse(B, 0)))
+(\)(L::Factor, B::SparseVecOrMat) = sparse(spsolve(CHOLMOD_A, L, Sparse(B)))
```

Comment:
It's not completely clear but the comment right above is the explanation for the zero so it is wrong to remove it here. Instead, I'd just split the method into one for matrices and one for vectors.